scripts/exp/ts-transform-import-path.js

```diff
@@ -49,45 +49,88 @@ export const importPathTransformer = () => ({
 
 function transformImportPath({ pattern, template }) {
   return context => file => {
-    return visitNodeAndChildren(file, context, { pattern, template });
+    const firstPass = visitNodeAndChildren(
+      file,
+      context,
+      { pattern, template },
+      transformDeclareNode
+    );
+    return visitNodeAndChildren(
+      firstPass,
+      context,
+      { pattern, template },
+      transformImportExportNode
+    );
```

Comment:
Can we do both in a single call? It seems the replacement logic is the same. We just need to update the condition to include module declaration?

Reply:
Updated to do a single pass. The culprit is the regex with the `-g` flag which is stateful. Fixed the issue by removing `-g`.